content/2016/10/18/versioning.rst

```diff
@@ -1,6 +1,8 @@
 public: no
 author: miriam
-tags: [Interviews, Sitepoint, Podcasts]
+tags: [Elsewhere, Interviews, SitePoint, Podcasts]
```

Comment:
Why don't podcasts show in our tag archive? https://deploy-preview-359--oddsite.netlify.com/tags/

Update: Looks like our other podcasts are under events. Should we display Podcast events also in articles?

Reply:
fixed